### PR TITLE
Download genomes based on NCBI taxonomy IDs

### PR DESCRIPTION
**Pull request by mgalardini**
Hi,

first of all, many thanks for this very useful piece of software. I have added two options to use the NCBI taxonomy IDs as filters instead of Genus/Species name:

* Species taxonomy ID to download all the members of a particular species (e.g. 562 for all E. coli)
* Taxonomy ID of a particular strain to download a specific genome (e.g. 511145 for "Escherichia coli str. K-12 substr. MG1655")

No tests are included for the moment, but I would be happy to do so if you would like to include this functionality.

Best,
Marco

### CODE REVIEW

**Comment by kblin** [comment]
Sure, this looks like a good idea. If you could fix the fake Namespaces in the tests and add some specific tests for the new parameters, I'm happy to merge this.

**Comment by mgalardini** [comment]
Sure thing, I'll get to it and let you know!

**Comment by mgalardini** [comment]
I have added two specific tests and updated the README with two examples of the new options

**Comment by kblin** [comment]
Looks good. I'll have a closer look once I'm back at my PC.
Thanks for the patch.

File: tests/test_core.py [hunk 1]
```diff
@@ -22,18 +22,18 @@ def test_download_one(monkeypatch, mocker):
     monkeypatch.setattr(core, '_download', _download_mock)
     fake_args = Namespace(section='refseq', domain='bacteria', uri=core.NCBI_URI,
                           output='/tmp/fake', file_format='genbank', assembly_level='all',
-                          genus='', parallel=1)
+                          genus='', species_taxid=None, taxid= None, parallel=1)
```
Comment (kblin) [inline tests/test_core.py]: please remove the space in `taxid= None`

**Comment by kblin** [comment]
Merged with a slight style fix at 724a2ecb2, thank you for your contribution.

**Comment by mgalardini** [comment]
No worries, thank you for this very nice piece of software!

**Pull request closed without merging**